lib/DidKeyDriver.js

```diff
@@ -168,8 +202,16 @@ export class DidKeyDriver {
    *   key pairs used to generate it (for storage in a KMS).
    */
   async _keyPairToDidDocument({keyPair} = {}) {
+    // get the multikeyHeader from the fingerprint
+    const multikeyHeader = getMultiKeyHeader({
+      fingerprint: keyPair.publicKeyMultibase || keyPair.publicKeyBase58
+    });
```

Comment:
hmm as far as I can tell it is not raw bytes. `fromFingerprint()` method returns a `publicKeyBase58` which is a base58 encoded string without the header - https://github.com/digitalbazaar/ed25519-verification-key-2018/blob/main/lib/Ed25519VerificationKey2018.js#L75-L79.  There is a `fingerprintFromPublicKey()` method which seems to be returning a multiformats encoded ed25519 public key https://github.com/digitalbazaar/ed25519-verification-key-2018/blob/main/lib/Ed25519VerificationKey2018.js#L216-L230. Is that what we can use in the adapter?

Reply:
From what you linked to and from this: https://github.com/digitalbazaar/ed25519-verification-key-2018/blob/main/tests/mock-data.js#L5 ... it's clear that `publicKeyBase58` doesn't include the multikey header. Using `fromFingerprint` seems like it would work fine in the adapter -- but we can't rely on any `publicKeyBase58` value for anything useful regarding a multibase-multikey header.

Reply:
Updated to get `publicKeyMultibase` from keypair if keypair contains`publicKeyBase58` [51c6b21](https://github.com/digitalbazaar/did-method-key/pull/55/commits/51c6b216ddd582cdbb4f85de80c34049ebb9b304).